src/main/java/org/fundacion/jala/converter/view/compiler/CompilerInterface.java

```diff
@@ -218,20 +210,20 @@ public int changeLanguage() {
     public void createProject() {
         LOGGER.info("start");
         ProjectRequestForm projectRequestForm = new ProjectRequestForm();
-        String result = (String) JOptionPane.showInputDialog(null,
+        String projectTitle = (String) JOptionPane.showInputDialog(null,
                 "Write project name", "Project Name",
                 JOptionPane.PLAIN_MESSAGE, null, null, "project1");
-        projectRequestForm.addProjectName(result);
-        projectRequestForm.addUserId("1");
-        try {
-            LOGGER.info("Execute Try");
-            String sResponse = clientRequest.executeRequest(projectRequestForm, token);
-            projectId = sResponse;
-            consoleOutput.getConsole().setText(sResponse + " " + result);
-            projectTab.start(extension);
-            LOGGER.info("finish");
-        } catch (Exception exception) {
-            LOGGER.error("Execute Exception" + exception.getMessage());
-        }
+            projectRequestForm.addProjectName(projectTitle);
+            projectRequestForm.addUserId("1");
+            try {
+                LOGGER.info("Execute Try");
+                String endpointResponse = clientRequest.executeRequest(projectRequestForm, token);
+                projectId = endpointResponse;
+                consoleOutput.getConsole().setText("Project " + projectTitle + " was created successfully");
+                projectTab.start(extension, projectTitle);
+                LOGGER.info("finish");
+            } catch (Exception exception) {
+                LOGGER.error("Execute Exception" + exception.getMessage());
+            }
```

Comment:
I Think the tab space in this lines, should be removed? doesn't look like the correct tabulation

Reply:
You're right, my mistake.